tool/tsh/config.go

```diff
@@ -42,7 +42,7 @@ Host *.{{ .ClusterName }} {{ .ProxyHost }}
 # Flags for all {{ .ClusterName }} hosts except the proxy
 Host *.{{ .ClusterName }} !{{ .ProxyHost }}
     Port 3022
-    ProxyCommand "{{ .TSHPath }}" proxy ssh --cluster={{ .ClusterName }} --proxy={{ .ProxyHost }} %r@%h:%p
+    ProxyCommand "{{ .TSHPath }}" proxy ssh --login=%r --cluster={{ .ClusterName }} --proxy={{ .ProxyHost }} %r@%h:%p
```

Comment:
Hm, I don't think this should be needed if we update the logic to take login from `tc.HostLogin` instead of `cf.NodeLogin` here: https://github.com/gravitational/teleport/blob/v8.0.0/tool/tsh/proxy.go#L125. Since the spec `%r@%h:%p` already includes the login. Am I missing something?

Reply:
Right, the `HostLogin` already contains the nodeLogin param. I have aligned the logic. Thanks.